### ISSUE TITLE
update docs

### ISSUE THREAD

**Issue by synkarius** [dictation-toolbox/Caster#131]
Lots of changes since v5.
- update Quick Reference guide
- update MD files
- annotate outdated YT videos

**Comment by Digital-ecologist** [comment]
Hey, @LexiconCode it's digital–ecologist from the KnowBrainer Speech Recognition form. It was a bit more work than I anticipated, but I was able to create that updated installation guide. It's not as polished as I would like it to be, but it should get somebody through the installation process on the first try. I wasn't able to test it from scratch as written, because I really only have my production machine to work with. I've attached the guide as a Word document. I can also get it into PDF or ODF format pretty easily. Thoughts, feedback and improvements are welcome. I hope this is helpful.
[new installation guide draft.docx](https://github.com/synkarius/caster/files/2377195/new.installation.guide.draft.docx)

**Comment by mrob95** [comment]
@Digital-ecologist I've only just seen this as it was buried at the bottom of the issues list, apologies. Looks good – I think I'm going to work on the docs a fair bit if I get time this term so will definitely incorporate this. It's unfortunate that it needs to be 20 lines long but hopefully we can get somewhere with simplifying the installation process.

**Comment by Digital-ecologist** [comment]
@mrob95 No worries, you and the team have done so much work, that the document I posted before is already moderately outdated. I just went through and tried to make the changes that reflect the current state of affairs. Here's a new version. I might know enough about Atom now to use it to convert the doc to markdown on Saturday, if that would be helpful, but now that I've got a functional voice-coding system for R, I really need to buckle down on finishing my thesis.
[Installation guide draft update 10.22.18.docx](https://github.com/synkarius/caster/files/2504307/Installation.guide.draft.update.10.22.18.docx) Best of luck improving the docs :-)
